frontend/js/src/brainzplayer/BrainzPlayerUI.tsx

```diff
@@ -219,7 +220,7 @@ function BrainzPlayerUI(props: React.PropsWithChildren<BrainzPlayerUIProps>) {
       <div className="actions">
         {isPlayingATrack && currentDataSourceName && (
           <a
-            href={trackUrl ?? "#"}
+            href={trackUrl || "#"}
```

Comment:
Out of curiosity, what was the need for this change?
Is it for when `trackUrl` is an empty string?

Reply:
Yup you got that right, it is for handling the empty string. I will also keep in mind to create separate PR for manage issues.